pages/cards/permutations.page.tsx

```diff
@@ -28,7 +28,14 @@ function createSimpleItems(count: number) {
 }
 
 const cardDefinition: CardsProps.CardDefinition<Item> = {
-  header: item => (item.number === 2 ? <Link href="#">{item.text}</Link> : item.text),
+  header: item =>
+    item.number === 2 ? (
+      <Link href="#" fontSize="heading-m">
```

Comment:
Would `fontSize="inherit"` make more sense? We probably didn't have that feature when we introduced the [website playground example](https://cloudscape.design/components/cards/?tabId=playground&example=simple).